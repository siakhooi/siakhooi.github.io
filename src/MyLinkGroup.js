import React from 'react'
import PropTypes from 'prop-types'

MyLinkGroup.propTypes = {
  children: PropTypes.array
}

function MyLinkGroup (props) {
  return (
    <div className='MyLinkGroup'>
      <ul className="MyLinkGroup">{props.children}</ul>
    </div>
  )
}
export default MyLinkGroup
