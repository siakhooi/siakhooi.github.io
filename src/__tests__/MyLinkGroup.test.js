import React from 'react'
import renderer from 'react-test-renderer'
import MyLinkGroup from '../MyLinkGroup'

it('MyLinkGroup/1', () => {
  const component =
    renderer.create(
      <MyLinkGroup><li>hello</li><li>hello</li></MyLinkGroup>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
