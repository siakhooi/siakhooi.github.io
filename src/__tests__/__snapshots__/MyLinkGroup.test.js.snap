// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyLinkGroup/1 1`] = `
<div
  className="MyLinkGroup"
>
  <ul
    className="MyLinkGroup"
  >
    <li>
      hello
    </li>
    <li>
      hello
    </li>
  </ul>
</div>
`;
